packages/page-pattern-modal/src/styles/page-pattern-modal.scss

```diff
@@ -102,6 +102,8 @@ $sidebar-width-desktop: 324px;
 .page-pattern-modal__inner {
 	@media screen and ( min-width: $breakpoint-mobile ) {
 		padding: 0 20px;
+		max-height: 100%;
+		display: flex;
```

Comment:
I think we might not need to set these two? And then we don't need to set the `width: 100%` to `page-pattern-modal__pattern-list-container` either. What do you think?

Reply:
Nice catch! I'll remove that.
I tested different approaches and now I am not sure why I ended up using these two.